tests/zeroconfig/windows/testdata/apps/AspNet.WebApi.NetFramework/Web.config

```diff
@@ -40,6 +40,10 @@
         <assemblyIdentity name="System.Buffers" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-4.0.5.0" newVersion="4.0.5.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.Numerics.Vectors" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-4.1.6.0" newVersion="4.1.6.0" />
```

Comment:
I'm kind of puzzled why there was no binding redirect before and now there is one. Anyway, it seems ok.